test/unit/math/fwd/fun/value_of_rec_test.cpp

```diff
@@ -0,0 +1,109 @@
+#include <stan/math/fwd.hpp>
+#include <stan/math/fwd/core.hpp>
+#include <gtest/gtest.h>
+#include <vector>
+
+TEST(AgradFwd, value_of_rec) {
+  using stan::math::fvar;
+
+  fvar<double> a = 5.0;
+  fvar<fvar<double> > ff_a(5.0);
+  fvar<fvar<fvar<fvar<double> > > > ffff_a(5.0);
```

Comment:
[optional] Change `> >` to `>>` in all this file.

Reply:
See above.